docs/business-apps/power-automate/guidance/working-with-get-items-and-get-files.md

```diff
@@ -12,36 +12,42 @@ The **Get items** and **Get files** SharePoint actions for flows in Power Automa
 **IMPORTANT:** First, and of primary consideration, the **Get items** action only works with lists, whereas the **Get files** action only works with libraries.
```

Comment:
Replace **IMPORTANT:** with [!IMPORTANT]

Reply:
Same as [!NOTE] - Seems like GitHub markdown doesn't recognize these?

Reply:
I removed the "Important" - I felt it is not necessary here as we are starting to educate users anyways.